roles/hammer_devel/tasks/hammer_install.yml

```diff
@@ -7,33 +7,43 @@
     remote: upstream
   with_items: "{{ hammer_devel_repositories }}"
 
-- name: 'Check if the fork remotes exist'
+- name: 'Check if the local fork remotes exist'
   shell: "git remote | grep ^{{ hammer_devel_github_fork_remote_name }}$"
   args:
     chdir: ~/{{ item.split('/')[1] }}
   ignore_errors: yes
+  changed_when: False
   with_items: "{{ hammer_devel_repositories }}"
-  register: fork_remotes_exist
+  register: local_fork_remotes_exist
 
-- name: 'Add fork remotes to cloned repositories'
-  command: "git remote add {{ hammer_devel_github_fork_remote_name }} git@github.com:{{ hammer_devel_github_username }}/{{ item.item.split('/')[1] }}.git"
-  when: item.rc != 0
+- name: 'Check the GitHub fork repos exist for missing local fork remotes'
+  shell: "git ls-remote --exit-code -h https://:@github.com/{{ hammer_devel_github_username }}/{{ item.item.split('/')[1] }}"
```

Comment:
Is the `:@` needed in the URL? Forces that to not ask for credentials on missing/private repos?

Reply:
Yep! It's interactive without the :@